java-client/src/main/java/org/apache/pegasus/client/ClientOptions.java

```diff
@@ -227,6 +227,11 @@ public boolean equals(Object options) {
     return false;
   }
 
+  @Override
+  public int hashCode() {
```

Comment:
Usually this is not a good practise to just call super.hashCode...

Reply:
Got it, i will fix it.